ingest-app/src/cmr/ingest/services/granule_bulk_update_service.clj

```diff
@@ -9,27 +9,69 @@
    [cmr.common.services.errors :as errors]
    [cmr.common.time-keeper :as time-keeper]
    [cmr.common.validations.json-schema :as js]
-   [cmr.ingest.data.bulk-update :as data-bulk-update]
+   [cmr.ingest.data.granule-bulk-update :as data-granule-bulk-update]
    [cmr.ingest.data.ingest-events :as ingest-events]
+   [cmr.common.log :as log :refer (debug info warn error)]
+   [cmr.ingest.services.bulk-update-service :as bulk-update-service]
    [cmr.ingest.services.ingest-service :as ingest-service]
    [cmr.transmit.metadata-db :as mdb]
    [cmr.transmit.metadata-db2 :as mdb2]
    [cmr.umm-spec.field-update :as field-update]))
 
+(def bulk-granule-update-schema
+ (js/json-string->json-schema (slurp (io/resource "granule_bulk_update_schema.json"))))
+
+(defmethod bulk-update-service/validate-bulk-update-post-params :granule
+ [_ json]
+ (js/validate-json! bulk-granule-update-schema json))
+
+(defn handle-bulk-update-event
```

Comment:
We are queuing bulk update event for each granule here, but I don't see anywhere that we are parsing out the bulk update instructions for each individual granules. I think it should be handled here.